docs/technical_reference/core/zero_order_unit.rst

```diff
@@ -0,0 +1,130 @@
+Zero Order Unit Model Base Class
+================================
+
+.. index::
+   pair: watertap.core.zero_order_unit;SITOBaseData
+
+.. currentmodule:: watertap.core.zero_order_unit
+
+The zero-order unit model base class is intended to form the basis for a library of generic single inlet-two outlet (SITO) units models using recovery and removal fractions to partition the incoming flow between the two outlets. This base class is not intended to be used by itself, and requires the user to define a derived class before it can be used in a flowsheet.
```

Comment:
```suggestion
The zero-order unit model base class is intended to form the basis for a library of generic single inlet-two outlet (SITO) unit models using recovery and removal fractions to partition the incoming flow between the two outlets. This base class is not intended to be used by itself and requires the user to define a derived class before it can be used in a flowsheet.
```